codegen-server/src/main/kotlin/software/amazon/smithy/rust/codegen/server/smithy/generators/protocol/ServerProtocolTestGenerator.kt

```diff
@@ -141,23 +239,25 @@ class ServerProtocolTestGenerator(
                 visibility = Visibility.PRIVATE,
             )
             writer.withModule(testModuleName, moduleMeta) {
-                renderAllTestCases(allTests)
+                renderAllTestCases(operationShape, operationSymbol, allTests)
             }
         }
     }
 
-    private fun RustWriter.renderAllTestCases(allTests: List<TestCase>) {
+    private fun RustWriter.renderAllTestCases(operationShape: OperationShape, operationSymbol: Symbol, allTests: List<TestCase>) {
```

Comment:
These methods do not need to take in `operationSymbol`: they can recompute it from the `operationShape` using the `symbolProvider`.

In general methods should not take in things which they can recompute from the state (class properties) and input parameters.